### PR TITLE
remove internal usage of solace_message

### PR DESCRIPTION
**Pull request by TamimiGitHub**

### CODE REVIEW

File: patterns/hello_world_pubsub.py [hunk 1]
```diff
@@ -106,12 +102,15 @@ def on_failed_publish(self, e: "FailedPublishEvent"):
...
+                .with_property("application", "samples") \
+                .with_property("language", "Python") \
+                .build(f'{message_body} --> {msgSeqNum}')
             # Direct publish the message
             direct_publisher.publish(destination=Topic.of(TOPIC_PREFIX + f"/python/{unique_name}/{msgSeqNum}"), message=outbound_msg)
             msgSeqNum += 1
-            # Modifying the outbond message instead of creating a new one
-            outbound_msg.solace_message.message_set_binary_attachment_string(f'{message_body} --> {msgSeqNum}')
-            outbound_msg.solace_message.set_message_application_message_id(f'sample_id {msgSeqNum}')
             time.sleep(0.1)
```
Comment (cjwmorgan-sol) [inline patterns/hello_world_pubsub.py]: time.sleep should not be necessary when using back the default back pressure elastic. Why is this here?

Comment (TamimiGitHub) [reply patterns/hello_world_pubsub.py]: I had it for demo purposes to show better show the flow of messages. I have however commented it out but kept it in case

File: patterns/hello_world_pubsub.py [hunk 2]
```diff
@@ -106,13 +107,15 @@ def on_failed_publish(self, e: "FailedPublishEvent"):
...
-            # Direct publish the message
-            direct_publisher.publish(destination=Topic.of(TOPIC_PREFIX + f"/python/{unique_name}/{msgSeqNum}"), message=outbound_msg)
             msgSeqNum += 1
-            # Modifying the outbond message instead of creating a new one
-            outbound_msg.solace_message.message_set_binary_attachment_string(f'{message_body} --> {msgSeqNum}')
-            outbound_msg.solace_message.set_message_application_message_id(f'sample_id {msgSeqNum}')
-            time.sleep(0.1)
+            # Check https://docs.solace.com/API-Developer-Online-Ref-Documentation/python/source/rst/solace.messaging.config.solace_properties.html for additional message properties
+            additional_properties = {APPLICATION_MESSAGE_ID: f'sample_id {msgSeqNum}'}
+            # Creating a dynamic outbond message
```
Comment (cjwmorgan-sol) [inline patterns/hello_world_pubsub.py]: typo of  `outbund`?

File: patterns/hello_world_pubsub.py [hunk 3]
```diff
@@ -106,13 +107,15 @@ def on_failed_publish(self, e: "FailedPublishEvent"):
...
-            direct_publisher.publish(destination=Topic.of(TOPIC_PREFIX + f"/python/{unique_name}/{msgSeqNum}"), message=outbound_msg)
             msgSeqNum += 1
-            # Modifying the outbond message instead of creating a new one
-            outbound_msg.solace_message.message_set_binary_attachment_string(f'{message_body} --> {msgSeqNum}')
-            outbound_msg.solace_message.set_message_application_message_id(f'sample_id {msgSeqNum}')
-            time.sleep(0.1)
+            # Check https://docs.solace.com/API-Developer-Online-Ref-Documentation/python/source/rst/solace.messaging.config.solace_properties.html for additional message properties
+            additional_properties = {APPLICATION_MESSAGE_ID: f'sample_id {msgSeqNum}'}
+            # Creating a dynamic outbond message 
+            outbound_message = message_builder.build(f'{message_body} --> {msgSeqNum}', additional_message_properties=additional_properties)
```
Comment (cjwmorgan-sol) [inline patterns/hello_world_pubsub.py:115]: Might be nice to comment that applciation id is override from the value of set in the builder?

Comment (TamimiGitHub) [reply patterns/hello_world_pubsub.py:115]: yeah that's a good idea 👍 will add it now

**Comment by TamimiGitHub** [comment]
I'll go ahead and merge this + add a 1.2 tag to master
